src/components/info/location-info.vue

```diff
@@ -0,0 +1,193 @@
+import { Coordinate } from 'ol/coordinate'
+import useMap from '@/composables/map/map.composable'
+import {
+  getQRUrl,
+  getElevation,
+  getNearestAddress,
+  queryInfo,
+} from '@/services/info/location-info'
+
+import StreetView from '@/components/info/street-view.vue'
+
+const { t } = useTranslation()
+const { locationInfo } = storeToRefs(useInfoStore())
+
+const map = useMap().getOlMap()
+
+const shortUrl = ref()
+const qrUrl = ref()
+const elevation = ref()
+const address = ref()
+const clickCoordinateLuref = ref()
+const formatted_coordinates = ref({})
+
+// initial load of position infos if locationInfo is not initially undefined
+// async function, no need to await the completion, DOM will be updated via refs
+updateInfo(locationInfo.value)
+
+watch(locationInfo, updateInfo)
+
+async function updateInfo(location: Coordinate | undefined) {
+  if (location) {
+    const info = await queryInfo(location, map.getView().getProjection())
+    shortUrl.value = info.shortUrl
+    qrUrl.value = getQRUrl(info.shortUrl)
+    clickCoordinateLuref.value = info.clickCoordinateLuref
+    formatted_coordinates.value = info.formatted_coordinates
+    elevation.value = await getElevation(info.clickCoordinateLuref)
+    address.value = await getNearestAddress(info.clickCoordinateLuref)
+  }
+}
+
+const isRapportForageVirtuelAvailable = computed(() => {
+  const userRole = 'ACT'
+  return userRole === 'ACT'
+})
+const downloadingRepport = ref(false)
+const isInBoxOfLidar = computed(() => true)
+const isCyclomediaAvailable = computed(() => true)
+const isImagesObliquesAvailable = computed(() => true)
+
+const getLidarUrl = () => 'bla'
+const cyclomediaUrl = computed(() =>
+  clickCoordinateLuref.value
+    ? `http://streetsmart.cyclomedia.com/streetsmart?q=${clickCoordinateLuref.value[0]};${clickCoordinateLuref.value[1]}`
+    : ''
+)
+const imagesObliquesUrl = computed(() =>
+  clickCoordinateLuref.value
+    ? `https://oblique.geoportail.lu/publication/viewer?x=${clickCoordinateLuref.value[0]}&y=${clickCoordinateLuref.value[1]}&crs=2169`
+    : ''
+)
+
+const addRoutePoint = () => '// TODO: add point to route'
+
+const open = ref(true)
+const { isStreetviewActive } = storeToRefs(useInfoStore())
+const toggleStreetview = () => {
+  isStreetviewActive.value = !isStreetviewActive.value
+}
+
+function downloadRapportForageVirtuel() {
+  downloadingRepport.value = true
+  setTimeout(() => (downloadingRepport.value = false), 2000)
+}
+</script>
+
+<template>
+  <div class="flex flex-row">
+    <div class="grow-[2] flex flex-col content-end">
+      <div class="grow"></div>
+      <h3 class="text-3xl text-white">
+        {{ t('Short Url', { ns: 'client' }) }}
+      </h3>
+      <input
+        onclick="this.select();"
+        class="lux-input"
+        type="text"
+        :value="shortUrl"
+        readonly="true"
+      />
+    </div>
+    <img class="mx-5 mt-5" v-if="qrUrl" :src="qrUrl" />
+  </div>
+  <div>
+    <div class="mt-5">
```

Comment:
looks like you can get rid of this dev and add the "mt-5" class to the h3 below.